orbit/utils/dataset.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
+
+def load_iclaims():
+    """
+    Load iclaims dataset
+
+    Returns
+    -------
+        pandas DataFrame
+
+    Notes
+    -----
+    https://fred.stlouisfed.org/series/ICNSA
```

Comment:
Also, mention google trend API for the regressors

Reply:
addressed